test/connector/http/generic/docker-compose.yml

```diff
@@ -32,4 +32,4 @@ services:
   dev:
     image: secretless-dev
     volumes:
-      - ../..:/secretless
+      - ../../../..:/secretless
```

Comment:
Per slack convo we should at some point figure this out - it's needlessly complicated of a path